src/NexusMods.App.UI/Pages/LoadoutPage/LoadoutView.axaml

```diff
@@ -28,11 +28,15 @@
                                    Title="All"
                                    Description="A combined view of all your installed mods from all collections." />
         </Panel>
+
+        <StackPanel Grid.Row="1">
+            <controls:StandardButton x:Name="ButtonUploadCollectionRevision" Text="Upload" />
```

Comment:
This should probably be hidden behind a feature flag. Either only available in debug mode or only available behind a setting

Reply:
This is done in the codebehind 👍